docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3'
+
+networks:
+    net:
+        name: net
+
+volumes:
+    aica:
+
+services:
+    db:
+        image: postgresql
+
+        build:
+            context: ./backend/docker
+            dockerfile: db.Dockerfile
```

Comment:
This dockerfile is meant for development only, as it creates a adminjs postgres user with a static password.